pkg/handlers/syncResources.go

```diff
@@ -91,6 +96,10 @@ func SyncResources(w http.ResponseWriter, r *http.Request) {
 	subscriptionUIDMap := make(map[string]bool) // map to hold exisiting subscription uids
 	response := SyncResponse{Version: config.AGGREGATOR_API_VERSION}
 
+	// !!! DO NOT APPROVE OR MERGE WITH THIS !!!
+	// Adding delay to test.
+	time.Sleep(6 * time.Minute)
```

Comment:
To remove?

Reply:
Removed with latest commit. I used this for testing to force the request to take long.